tests/common.py

```diff
@@ -41,3 +41,17 @@ def get_output(doc):
             output[identifier_el.text] = data_el[0].text
 
     return output
+
+
+def run_wps_process(process, params):
+    client = client_for(Service(processes=[process]))
+    datainputs = params
+    resp = client.get(
+        service="wps",
+        request="Execute",
+        version="1.0.0",
+        identifier=process.identifier,
+        datainputs=datainputs,
+    )
+
+    assert_response_success(resp)
```

Comment:
I seem to be failing on local `pytest` runs. Are there any additional installation steps I should be aware of aside from the usual pip installations?

Reply:
I have edited the instruction to resolve the RVIC issue above.